test/e2e/jest.config.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
   setupFiles: ['./test/config/copy.bpmn.diagram.ts'],
   setupFilesAfterEnv: [
     'expect-playwright',
+    './test/config/jest.setup.after.env.ts',
```

Comment:
❓ can we rename the file into jest.retries.ts?

Reply:
I didn't know if we wanted to add other configuration here, but ok to renamed it 🙂